### PR TITLE
Fortify - Make module dynamically check config for presets

### PR DESCRIPTION
**Pull request by Freddo3000**
**When merged this pull request will:**
- Make the fortify module check missionConfigFile and configFile, instead of being hardcoded.

### CODE REVIEW

File: addons/fortify/Cfg3DEN.hpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Cfg3DEN
+{
```
Comment (mharis001) [inline addons/fortify/Cfg3DEN.hpp:2]: Opening braces should be on same line as `class`.

File: addons/fortify/Cfg3DEN.hpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                class Value;
+            };
+        };
+        class GVAR(presetSelection): Combo {
+            class Controls: Controls {
+                class Title: Title {};
+                class Value: Value {
+                    delete Items;
+                    class ItemsConfig {
+                        path[] = { QGVAR(Presets) };
```
Comment (mharis001) [inline addons/fortify/Cfg3DEN.hpp:18]: No spaces around braces.
```suggestion
                        path[] = {QGVAR(Presets)};
```

File: addons/fortify/Cfg3DEN.hpp [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            };
+        };
+        class GVAR(presetSelection): Combo {
+            class Controls: Controls {
+                class Title: Title {};
+                class Value: Value {
+                    delete Items;
+                    class ItemsConfig {
+                        path[] = {QGVAR(Presets)};
+                        localConfig = 1;
```
Comment (PabstMirror) [inline addons/fortify/Cfg3DEN.hpp:17]: this system is really nice

File: addons/fortify/Cfg3DEN.hpp [hunk 4]
```diff
@@ -0,0 +1,25 @@
+class Cfg3DEN {
+    class Attributes {
+        class Title;
+        class Combo: Title {
+            class Controls {
```
Comment (PabstMirror) [inline addons/fortify/Cfg3DEN.hpp:5]: `Updating base class Controls->, by z\acex\addons\fortify\config.cpp/Cfg3DEN/Attributes/Combo/Controls/ (original (a3\3den\config.bin - no unload))`

**Pull request merged**